.prettierignore

```diff
@@ -1,4 +1,3 @@
 node_modules
 .rollup.cache
 dist
-tsconfig.json
```

Comment:
I think moon was formatting this file. what happens now ?

Reply:
Seen together, we let `Prettier` be the source of the truth for formatting files.